src/data/storageService.js

```diff
@@ -0,0 +1,65 @@
+import { S3Client, PutObjectCommand, GetObjectCommand } from '@aws-sdk/client-s3'
```

Comment:
Should this file be added to api.js so all of my function calls in the React app only reference one source? Or kept separate to separate functionality?

Reply:
I would leave it separate since it abstracts the storage implementation from the rest of the application. Within the storageService.js file itself, you could potentially swap out S3 with Azure blobs without needing to make any other changes to the rest of the app.